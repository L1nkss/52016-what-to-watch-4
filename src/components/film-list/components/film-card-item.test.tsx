import * as ShallowRenderer from 'react-test-renderer/shallow';
import FilmCardItem from "@components/film-list/components/film-card-item";

const film = {
  name: `Fantastic Beasts: The Crimes of Grindelwald`,
  previewImage: `fantastic-beasts-the-crimes-of-grindelwald`,
  previewVideoLink: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
  id: 2
};

describe(`Testing film card item component`, () => {
  it(`Component should successfully rendered`, () => {
    const myShallowRenderer = ShallowRenderer.createRenderer();
    myShallowRenderer.render(<FilmCardItem data={film}/>);
    const result = myShallowRenderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
