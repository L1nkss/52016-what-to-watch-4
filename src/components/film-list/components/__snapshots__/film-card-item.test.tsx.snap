// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing film card item component Component should successfully rendered 1`] = `
<WithVideoPlayer
  filmInfo={
    Object {
      "id": 2,
      "name": "Fantastic Beasts: The Crimes of Grindelwald",
      "previewImage": "fantastic-beasts-the-crimes-of-grindelwald",
      "previewVideoLink": "https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4",
    }
  }
/>
`;
